vscode-graphql language service server fails to start after extension update

I updated my vscode-graphql extension today and the language service server fails with the following error log:
```
8/11/2022, 3:16:06 PM [3] (pid: 24897) graphql-language-service-usage-logs: {"type":"usage","messageType":"initialize"}
8/11/2022, 3:16:06 PM [1] (pid: 24897) graphql-language-service-usage-logs: WARNING: graphql-config error, only highlighting is enabled:
Cannot find module '@cspotcode/source-map-support'
Require stack:
- /Users/parker/.vscode-insiders/extensions/graphql.vscode-graphql-0.7.2/out/server/index.js
for more information on using 'graphql-config' with 'graphql-language-service-server', 
see the documentation at https://www.npmjs.com/package/graphql-language-service-server#user-content-graphql-configuration-file
8/11/2022, 3:16:06 PM [1] (pid: 24897) graphql-language-service-usage-logs: WARNING: graphql-config error, only highlighting is enabled:
Cannot find module '@cspotcode/source-map-support'
Require stack:
- /Users/parker/.vscode-insiders/extensions/graphql.vscode-graphql-0.7.2/out/server/index.js
for more information on using 'graphql-config' with 'graphql-language-service-server', 
see the documentation at https://www.npmjs.com/package/graphql-language-service-server#user-content-graphql-configuration-file
8/11/2022, 3:16:06 PM [1] (pid: 24897) graphql-language-service-usage-logs: WARNING: graphql-config error, only highlighting is enabled:
Cannot find module '@cspotcode/source-map-support'
Require stack:
- /Users/parker/.vscode-insiders/extensions/graphql.vscode-graphql-0.7.2/out/server/index.js
for more information on using 'graphql-config' with 'graphql-language-service-server', 
see the documentation at https://www.npmjs.com/package/graphql-language-service-server#user-content-graphql-configuration-file
```
The graphqlrc has not changed since before the upgrade, but this is the config:
```
projects:
    default:
        schema:
            - './packages/snclient-graphql-types/src/generated/schema.graphql'
            - './packages/snclient-graphql-types/src/DevelopmentMocks.graphql'
        extensions:
            codegen:
                generates:
                    packages/graphql-types/src/generated/graphqlTypings.ts:
                        schema:
                            - https://dev.example.com/graphql:
                                headers:
                                    Authorization: Bearer ${API_ACCESS_TOKEN}
                        plugins:
                            - typescript
                            - add:
                                content: "import {JSON} from '../CustomScalars'"
                            - add:
                                content: "export type UnMaybe<T> = Exclude<T, undefined | null>;"
                        config:
                            scalars:
                                After: string
                                Before: string
                                DirectionTreeInput: JSON
                                DistributionMetricDataInput: JSON
                                # A field whose value matches the provided regular expression /^1$/.
                                FacebookBusinessCAMV1Version: string
                                Filter: string
                                First: number
                                # A field whose value matches the provided regular expression /^1$/.
                                InstagramFeedV1Version: string
                                # A field whose value matches the provided regular expression /^1$/.
                                InstagramStorySegmentV1Version: string
                                JSON: JSON
                                Last: number
                                OrderBy: string
                                OrderDataInput: JSON
                                OrderDir: "'asc' | 'desc'"
                                ProductionTreeInput: JSON
                                Search: string
                                SocialAccountCreateInput: JSON
                                UnixTimeSec: number

                    packages/graphql-types/src/generated/possibleTypes.ts:
                        plugins:
                            - 'fragment-matcher'
                    packages/graphql-types/src/generated/schema.graphql:
                        plugins:
                            - 'schema-ast'
                overwrite: true
```

Environment:
```
Extension: graphql.vscode-graphql v0.7.2
Version: 1.71.0-insider (Universal)
Commit: dced70bbf36d3c53c08e791da1791ac7fc42519b
Date: 2022-08-10T05:18:38.020Z
Electron: 19.0.11
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 20.6.0
Sandboxed: Yes
```

--- a/.changeset/six-yaks-heal.md
+++ b/.changeset/six-yaks-heal.md
@@ -1,0 +1,6 @@
+---
+'graphql-language-service-server': patch
+'vscode-graphql': patch
+---
+
+This reverts the bugfix for .graphqlrc.ts users, which broke the extension for schema url users

--- a/custom-words.txt
+++ b/custom-words.txt
@@ -68,7 +68,6 @@ codemirror
 codesandbox
 commitlint
 cosmicconfig
-cspotcode
 dompurify
 dset
 esbuild

--- a/package.json
+++ b/package.json
@@ -138,6 +138,6 @@
     "wsrun": "^5.2.4"
   },
   "resolutions": {
-    "graphql-config": "4.3.3"
+    "graphql-config": "4.3.0"
   }
 }

--- a/packages/graphql-language-service-server/package.json
+++ b/packages/graphql-language-service-server/package.json
@@ -37,7 +37,7 @@
     "@babel/types": "^7.17.0",
     "@babel/parser": "^7.17.3",
     "dotenv": "8.2.0",
-    "graphql-config": "4.3.3",
+    "graphql-config": "4.3.0",
     "graphql-language-service": "^5.0.6",
     "mkdirp": "^1.0.4",
     "node-fetch": "^2.6.1",

--- a/packages/graphql-language-service/package.json
+++ b/packages/graphql-language-service/package.json
@@ -44,10 +44,10 @@
     "@types/json-schema": "7.0.9",
     "benchmark": "^2.1.4",
     "graphql": "^16.4.0",
-    "graphql-config": "4.3.3",
+    "graphql-config": "4.3.0",
     "lodash": "^4.17.15",
     "platform": "^1.3.5",
-    "ts-node": "^10.8.1",
+    "ts-node": "^8.10.2",
     "typescript": "^4.6.3"
   },
   "scripts": {

--- a/packages/vscode-graphql/esbuild.js
+++ b/packages/vscode-graphql/esbuild.js
@@ -11,11 +11,7 @@ build({
   minify: arg === '--minify',
   platform: 'node',
   outdir: 'out/',
-  external: [
-    'vscode',
-    '@cspotcode/source-map-support', // workaround for https://github.com/cspotcode/node-source-map-support/issues/43
-    './transpilers/swc.js', // suppresses warning via ts-node, but we won't be using it, so we can exclude it
-  ],
+  external: ['vscode'],
   format: 'cjs',
   sourcemap: true,
   watch: isWatchMode,

